### PR TITLE
implemented Upload new CV modal(pop-up) #30

### PR DESCRIPTION
**Pull request by KavyaReddy732**
# Description
implemented the cv model  
Please provide a short summary explaining what this PR is about.
uploadCv
Fixes # (issue)

# How to test?
this can't see this model in storybook because story book doesn't accept react dom elements.
import the component to App.js and it will appear on screen without disturbing any functionality. 
Please provide a short summary how your changes can be tested?

# Checklist

- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by benna100 — CHANGES_REQUESTED** [review]
There are some error you need to fix! FX line 3 in addResume.js

File: src/client/components/AddResume/AddResume.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+.popup-form{
+    padding: 20px;
+    border: 1px solid rgb(218, 214, 214);
+    border-radius: 10px;
+    height: 500px;
+    width: 700px;
+}
+.closeicon{
```
Comment (benna100) [inline src/client/components/AddResume/AddResume.css:16]:
```suggestion
.close-icon{
```

File: src/client/components/AddResume/AddResume.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+.reset-uplod-btns button{
+    height: 30px;
+    width: 100px;
+    margin:4px;
+   padding: 0px;  
+}
+
+.reset-uplod-btns .reset-btn button{
```
Comment (benna100) [inline src/client/components/AddResume/AddResume.css:79]:
```suggestion
.reset-upload-btns .reset-btn button{
```

File: src/client/components/AddResume/AddResume.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+export const AddResume = () => {
+const[title,setTitle] = useState('');
+const[description,setDescription] = useState('');
+const[uploadFile,setUploadaFile] =useState('');
+
+    return ReactDom.createPortal ((
+        <div className = "popup">
+            <div className = "popup-form">
+                <img className="closeicon" src= {close} alt='closeIcon'/>
```
Comment (benna100) [inline src/client/components/AddResume/AddResume.js:15]: Human readable alt attribute

```suggestion
                <img className="closeicon" src= {close} alt='Close icon'/>
```

File: src/client/components/AddResume/AddResume.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+                <h3>Upload new CV</h3>
+            <div className='form'>
+                <div className = "title-box">
+                    <label>Add Title</label><br/>
+                    <input type='text' value={title} onChange={(e)=> setTitle(e.target.value)}/><br/>
+                </div>
+                <div className = "description-box">
```
Comment (benna100) [inline src/client/components/AddResume/AddResume.js:22]: I dont know how much information is in the box name!?

```suggestion
                <div className="description">
```

File: src/client/components/AddResume/AddResume.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+                    <textarea type='text' value={description} onChange={(e)=> setDescription(e.target.value)}/><br/>
+                </div>
+                <div className = "upload-box">
+                    <label>Select A file</label><br/>
+                    <input type='text' value={uploadFile} onChange={(e)=> setUploadaFile(e.target.value)}/>
+                    <Button buttonName='Browse' backgroundColor = {("backgroundColor", "black")} /><br/>
+                </div>
+            </div>
+            <div className = "reset-uplod-btns">
+                    <span className = 'reset-btn'><Button buttonName='Reset' backgroundColor = {("backgroundColor", "white")} /></span>
```
Comment (benna100) [inline src/client/components/AddResume/AddResume.js:33]: Looks a bit weird. Do you need the span wrapper?

File: src/client/components/AddResume/AddResume.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+                    <span className = 'upload-btn'><Button buttonName='Upload' backgroundColor = {("backgroundColor", "#0388fc")} /></span><br/><br/>
+                <div className = "add-btn">
+                    <Button buttonName='Add resume' backgroundColor = {("backgroundColor", "rgb(233, 86, 29)")} />
+                </div>
+            </div>
+        </div>    
+    </div>
+    ), document.getElementById('portal')
```
Comment (benna100) [inline src/client/components/AddResume/AddResume.js:41]:
```suggestion
    ), document.querySelector('#portal')
```

**Review by benna100 — CHANGES_REQUESTED** [review]
Very strong work Kavya!

File: src/client/components/AddResume/AddResume.js (hunk 5 above)
Comment (KavyaReddy732) [reply src/client/components/AddResume/AddResume.js:33]: i want those two buttons in same line. i can use 'div' but i need more styling i.e using flex or grid to get them in same line. so, i used span wrapper.

Comment (KavyaReddy732) [reply src/client/components/AddResume/AddResume.js:33]: if you want me to change that.i can change it to div.

File: src/client/components/AddResume/AddResume.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+import './AddResume.css';
+import close from '../../assets/images/closeIcon.svg';
+
+export const AddResume = () => {
+const[title,setTitle] = useState('');
+const[description,setDescription] = useState('');
+const[uploadFile,setUploadaFile] =useState('');
+
+    return ReactDom.createPortal ((
+        <div className = "popup">
```
Comment (benna100) [inline src/client/components/AddResume/AddResume.js:13]: You have lots of spaces, please fix that

```suggestion
        <div className="popup">
```

File: src/client/components/AddResume/AddResume.js (hunk 5 above)
Comment (benna100) [reply src/client/components/AddResume/AddResume.js:33]: Its fine, no worries

**Review by benna100 — CHANGES_REQUESTED** [review]
Can you please fix all the spacings, then it is ready to merge!

**Review by benna100 — APPROVED** [review]
Looks good! Well done kavya!

**Pull request merged**